packages/mjml-text/README.md

```diff
@@ -19,7 +20,7 @@ This tag allows you to display text in your email.
  ```
 
 <p style="text-align: center;" >
-  <a href="https://mjml.io/try-it-live/components/text">
+  <a href="https://mjml.io/try-it-live/E7Tk_8zjI">
```

Comment:
You don't need to change the link here it will fetch the content automatically 👍